addons/packages/kpack/0.5.0/README.md

```diff
@@ -0,0 +1,565 @@
+# kpack
+
+[kpack](https://github.com/pivotal/kpack) utilizes unprivileged Kubernetes primitives to provide builds of OCI images as a platform implementation of [Cloud Native Buildpacks](https://buildpacks.io) (CNB).
+
+## Components
+
+* kpack
+
+## Supported Providers
+
+The following tables shows the providers this package can work with.
+
+ | AKS | EKS | vSphere | Docker |
+|-----|-----|---------|--------|
+| ✅   | ✅   | ✅       | ✅      |
```

Comment:
AKS and EKS would mean we are referring to the managed K8s clusters provided by Azure and AWS respectively. Since TCE is more of a self hosted K8s cluster, I think we can replace `AKS` with `Azure` and `EKS` with `AWS`, something like this https://github.com/vmware-tanzu/community-edition/blob/8ce93fa92e47d355e4d0ebbc3e18c7fd703f6bbd/addons/packages/harbor/2.2.3/README.md?plain=1#L8-L10

Reply:
the supported providers for the other readme list 

AWS | Azure | vSphere | Docker





<!--EndFragment-->
</body>
</html>